### ISSUE TITLE
[Support]: How I could see in the Web UI that a camera is disconnected/online/offline without restarting frigate?

### ISSUE THREAD

**Issue by heresiarch** [blakeblackshear/frigate#4945]
### Describe the problem you are having

When I look on my cameras, there are cameras which are disconnected but I could see still the latest Picture before disconnect. This is good to access historical recordings/events but I would like to see in the UI an health indicator, that the camera is online or offline. A simple red or green bubble to show that I'm really looking on the live picture.
I know there are MQTT events for this, but it would be good to have in the camera overview and birdseye view an clear indicator on and offline state.
When I restart frigate the cameras show a green frame but not if the cameras disconnected and frigate is not restarted.

### Version

Debug 0.11.1-2eada21

### Frigate config file

```yaml
mqtt:
  host: 192.168.178.8
  port: 8883
  topic_prefix: frigate
  client_id: frigate
  tls_ca_certs: /config/ca.crt
  tls_client_cert: /config/client.crt
  tls_client_key: /config/client.key
  tls_insecure: true
 
detectors:
  coral:
    type: edgetpu
    device: usb
cameras:
  steinel:
    ffmpeg:
      inputs:
        - path: rtsp://192.168.178.76:554/user=admin&password=XXX&channel=1&stream=0.sdp?real_stream--rtp-caching=500
          roles:
            - rtmp
            - record
        - path: rtsp://192.168.178.76:554/user=admin&password=XXX&channel=1&stream=1.sdp?real_stream--rtp-caching=500
          roles:
            - detect
      output_args:
        record: -map 0:v -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c:v h264_qsv -b:v 4M -r 15
    detect:
      width: 640 # <---- update for your camera's resolution
      height: 360 # <---- update for your camera's resolution
  camera_1: 
    ffmpeg:
      inputs:
        - path: rtsp://camadmin:XXX@192.168.178.31/stream1
          roles:
            - rtmp
            - record
        - path: rtsp://camadmin:XXX@192.168.178.31/stream2
          roles:
            - detect    
    detect:
      width: 640 # <---- update for your camera's resolution
      height: 360 # <---- update for your camera's resolution
  camera_2: 
    ffmpeg:
      inputs:
        - path: rtsp://camadmin:XXX@192.168.178.30/stream1
          roles:
            - rtmp
            - record
        - path: rtsp://camadmin:XXX@192.168.178.30/stream2
          roles:
            - detect   
    detect:
      width: 640 # <---- update for your camera's resolution
      height: 360 # <---- update for your camera's resolution 

birdseye:
  enabled: True
  mode: continuous

record:
  enabled: True
  retain:
    days: 7
    mode: motion
  events:
    retain:
      default: 14
      mode: active_objects


ffmpeg:
  hwaccel_args: -hwaccel qsv -qsv_device /dev/dri/renderD128

rtmp:
  # Optional: Enable the RTMP stream (default: True)
  enabled: False

logger:
 # Optional: default log level (default: shown below)
  default: debug
  logs:
    frigate.app: debug
    frigate.mqtt: debug
    frigate.edgetpu: debug
    frigate.zeroconf: debug
```

### Relevant log output

```shell
not relevant
```

### FFprobe output from your camera

```shell
not relevant
```

### Operating system

Debian

### Install method

Docker Compose

### Coral version

USB

### Network connection

Wireless

### Camera make and model

x264 streams with Tapo C320WS and HEVC Steinel L 620 CAM

**Comment by blakeblackshear** [comment]
This has been addressed in 0.12.0 beta1

**Comment by heresiarch** [comment]
Thank you very much I will try.
